google/ggrc-core
# GGRC 6322 Make objects mapped to WF readable by users with WF role

# Issue description

Users who are a workflow members/admins or cycle task assignees does not see objects mapped to cycle task if they don't have access to them, but they should.

# Steps to test the changes

1. Create a workflow with a task;
2. Activate workflow;
3. Map some objects to the cycle task;
4. Add a global creator as a workflow admin / workflow member / cycle task assignee / etc.;
5. Open a cycle task as this another global creator;
6. Mapped objects should be seen.

# Solution description

Extend propagation tree with read permissions on objects mapped to "CycleTaskGroupObjectTask" for users having "Admin" , "Task Assignees", "Task Secondary Assignees" or "Workflow Member" role.

# Sanity checklist


# Migration checklist


# PR Review checklist
